example/CMakeLists.txt

```diff
@@ -1,5 +1,11 @@
 project(capo-example)
+set(cxx_standard cxx_std_20)
 
-add_executable(${PROJECT_NAME})
-target_link_libraries(${PROJECT_NAME} PRIVATE capo::capo capo::options)
-target_sources(${PROJECT_NAME} PRIVATE example.cpp)
+add_executable(${PROJECT_NAME}-version)
+target_link_libraries(${PROJECT_NAME}-version PRIVATE capo::capo capo::options)
+target_sources(${PROJECT_NAME}-version PRIVATE version.cpp)
+
+add_executable(${PROJECT_NAME}-openal-sound)
```

Comment:
I was thinking of building up example itself; do you have another plan in mind, with multiple executables?

Reply:
I wasn't sure what you were going for with the example so I didn't want to risk removing what you already had -- I'll integrate the sound.cpp code into version.cpp and revert my changes

Reply:
Ah okay! I think while we're experimenting it's fine to do anything, I'd just named the "playground" `example` because there will eventually be one.